linux_os/guide/system/software/integrity/certified-vendor/installed_OS_is_FIPS_certified/rule.yml

```diff
@@ -1,6 +1,6 @@
 documentation_complete: true
 
-prodtype: rhel6,rhel7,rhel8,fedora,ol7,rhv4
+prodtype: rhel6,rhel7,rhel8,fedora,ol7,rhv4, ol8, wrlinux1019
```

Comment:
I think `prodtype` can be `all`, so that this rule can be include in every product, and fail if not FIPS certified / compliant.